modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -824,6 +826,16 @@ public ObjectDefinition putObjectDefinitionByExternalReferenceCode(
 		return postObjectDefinition(objectDefinition);
 	}
 
+	private Map<String, String> _addDefaultLocaleLabel(
+		String defaultLocale, Map<String, String> localeLabelMap, String name) {
+
+		if (!localeLabelMap.containsKey(defaultLocale)) {
+			localeLabelMap.put(defaultLocale, name);
+		}
+
+		return localeLabelMap;
+	}
```

Comment:
We don't need this method. We can use computeIfAbsent() instead of this method in its usage.